waspc/data/Generator/templates/sdk/wasp/server/auth/hooks.ts

```diff
@@ -21,6 +21,16 @@ export type OnBeforeOAuthRedirectHook = (
   params: Expand<OnBeforeOAuthRedirectHookParams>,
 ) => { url: URL } | Promise<{ url: URL }>
 
+// PUBLIC API
+export type OnBeforeLoginHook = (
```

Comment:
Make sure to update the visible API table. I know this is annoying, I hope we can get rid of it soon.